packages/calypso-e2e/src/jest-playwright-config/environment.ts

```diff
@@ -271,6 +278,10 @@ class JestEnvironmentPlaywright extends NodeEnvironment {
 			case 'test_fn_failure': {
 				this.failure = { type: 'test', name: event.test.name };
 				this.allure?.failTestStep( event.error );
+				// Store the failing test's artifact path if it hasn't yet.
+				if ( this.testFailureArtifacts.indexOf( this.testArtifactsPath ) === -1 ) {
```

Comment:
**suggestion (non-blocking)**: I would recommend using the `includes()` method here because it's a bit more clear & obvious what it's checking (even though the index comparison was the go-to way for a while). 

In general, you'll find in Calypso that folks often take advantage of the newer syntax in TypeScript / newer ECMAScript standards!

Reply:
Sorry for belated reply, I would say your recent review overrides my requires change marker.

Reply:
Great suggestion, @dpasque! It's way cleaner and easier to understand now. Since this was a non-blocking suggestion, I'll merge the PR once the checks and test runs look good. Thanks again 😄